commit 9f294da169ca311bac2224893654cfe58fd81e4b
Author: Michael Cyr <mikecyr@us.ibm.com>
Date:   Fri Mar 17 00:48:20 2017 +0000

    scsi: ibmvscsis: Clean up properly if target_submit_cmd/tmr fails
    
    [ Upstream commit 7435b32e2d2fb5da6c2ae9b9c8ce56d8a3cb3bc3 ]
    
    Signed-off-by: Michael Cyr <mikecyr@us.ibm.com>
    Signed-off-by: Bryant G. Ly <bryantly@linux.vnet.ibm.com>
    Tested-by: Steven Royer <seroyer@linux.vnet.ibm.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1418_9f294da169ca311bac2224893654cfe58fd81e4b.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1418_9f294da169ca311bac2224893654cfe58fd81e4b.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
